### PR TITLE
solution

### PR DESCRIPTION
**Pull request by venher-v**
https://venher-v.github.io/react_goods-selector-js/

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -14,70 +15,66 @@ export const goods = [
...
-            >
-              -
-            </button>
-          </td>
+            return (
+              <tr
+                key={good}
+                data-cy="Good"
+                className={
+                  selected ? 'has-background-success-light' : ''
```
Comment (volodymyr-soltys97) [inline src/App.jsx:52]: Use a classnames library for add classes with condition

File: src/App.jsx [hunk 2]
```diff
@@ -14,70 +15,66 @@ export const goods = [
...
+                    onClick={() => setSelectedGood(selected ? '' : good)}
+                    data-cy={
+                      selected ? 'RemoveButton' : 'AddButton'
+                    }
+                    type="button"
+                    className={
+                      selected ? 'is-info button' : 'button'
+                    }
+                  >
+                    {selected ? '-' : '+'}
```
Comment (volodymyr-soltys97) [inline src/App.jsx:66]: The same as above

File: src/App.jsx [hunk 3]
```diff
@@ -14,70 +15,66 @@ export const goods = [
...
+              <tr
+                key={good}
+                data-cy="Good"
+                className={
+                  selected ? 'has-background-success-light' : ''
+                }
+              >
+                <td>
+                  <button
+                    onClick={() => setSelectedGood(selected ? '' : good)}
```
Comment (volodymyr-soltys97) [inline src/App.jsx:57]: Consider will be better if you move this logic to the helper function and use it here

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
Good job 👍 
Let's improve your code

File: src/App.jsx [hunk 4]
```diff
@@ -14,70 +16,73 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectedGood, setSelectedGood] = useState('Jam');
```
Comment (OleziO) [inline src/App.jsx:20]:
```suggestion
  const [selectedGood, setSelectedGood] = useState(goods.find(item => item === 'Jam') || goods[0]);
```

Its more safe variant.

**Review by OleziO — APPROVED** [review]

**Pull request closed without merging**